agent/php_execute.c

```diff
@@ -107,6 +107,12 @@ typedef void (*nr_framework_enable_fn_t)(TSRMLS_D);
  */
 typedef void (*nr_library_enable_fn_t)(TSRMLS_D);
 
+/*
+ * Purpose: Enable monitoring on specific functions for a detected vulnerability
+ *          management package.
+ */
+typedef void (*nr_vuln_mgmt_enable_fn_t)(TSRMLS_D);
```

Comment:
I don't believe `TSRMLS_D` is needed in new code that is added.

Reply:
Fixed: [bc8b469](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/bc8b4695646a148bc1ffd0a127d2d10126df8923)